Payments/Integration/01_ManageConnection.md

```diff
@@ -0,0 +1,140 @@
+
+4. Enter the required data in the fields in the *Credentials* section. Use the information you received from the payment service provider to fill the credentials.   
+5. If the PSP connection requires additional settings, open the *Settings* tab and add further settings.
+
+6. Click the [SAVE] button.   
+    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+     ![Checking credentials](../../Assets/Screenshots/Payments/Settings/CheckingCredentials.png "[Checking credentials]")
+
+    The view for creating connections is automatically closed when the connection has been established. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
+      ![Credentials](../../Assets/Screenshots/Payments/Settings/EditConnectionSettings.png "[Credentials]")
+    <!-----Stefan: stimmt das? Ich kann es nicht nachmachen--->
+
+
+## Edit a PSP connection
+
+Edit a connection to adjust any changed credentials, the connection name or further settings. 
+
+#### Prerequisites
+
+At least one connection has been created, see [Create a connection](#create-a-connection).
+
+#### Procedure
+
+*Payments > Settings > Tab CONNECTIONS*
+
+![Connections](../../Assets/Screenshots/Payments/Settings/ChangeConnections.png "[Connections]")
+
+1. Click the checkbox on the left to select the connection you want to edit.  
+   The editing toolbar is displayed above the list of connections.      
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") button
+   
+    The *Edit connection* view is displayed.
+
+    > [Info] Depending on the driver, the *Credentials* as well as the *Settings* tab are displayed in the *Edit connection* view.
+
+    ![Connection credentials](../../Assets/Screenshots/Payments/Settings/EditConnectionCredentials.png "[Connection credentials]")
+
+2. Edit the desired data of the connection in the corresponding fields in the *Credentials* tab.
+
+3. If necessary, click the *Settings* tab and edit the desired settings of the connection in the corresponding fields.
+
+4. Click the [SAVE] button.   
+    The *Checking credentials...* notice is displayed.
+    ![Checking credentials](../../Assets/Screenshots/Payments/Settings/CheckingCredentials.png "[Checking credentials]")
+    The *Edit connection* view is automatically closed when the changes have been saved. The *CONNECTIONS* view is displayed again.
+
+## Disable a PSP connection
+Once a connection has been established, you cannot delete it because of potential dependencies in the existing database. Nevertheless, you can disable the connection if you no longer use it. You can disable several active connections at a time.
+
+#### Prerequisites
+
+- At least one connection has been established, see [Create a connection](#create-a-connection).
+- The connection is active.
```

Comment:
At least one connection is active. 

Vgl. Disable prerequisites.

Reply:
Done